pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/gp/KitInfo.java

```diff
@@ -19,9 +19,10 @@ public class KitInfo {
     private String accessionNumber;
     private String realm;
     private String kitTypeName;
+    private String collectionDate;
 
     public KitInfo(String sampleCollectionBarcode, int organismClassificationId, String gender, String bspParticipantId, String bspSampleId,
-                   String materialInfo, String receptacleName, String realm, String kitTypeName) {
+                   String materialInfo, String receptacleName, String realm, String kitTypeName, String collectionDate) {
```

Comment:
Why is that change needed here?

Reply:
Because for clinical Kit route we also return KitInfo and we need to return collection date, I fixed it for BSP